server/inngest/eventHandlers/sendUpcomingNotifications.ts

```
@@ -0,0 +1,136 @@
+    const tomorrowStart = getBrazilTomorrowStart();
+    const tomorrowEnd = getBrazilTomorrowEnd();
+    
+    const subscriptionsToNotify = subscriptions.filter(sub => 
+      sub.schedule.startAt >= tomorrowStart && 
+      sub.schedule.startAt <= tomorrowEnd &&
+      !sub.notificationsUpcomingSentAt
+    );
+    
+    if (!subscriptionsToNotify.length) return { skipped: true, reason: "No subscriptions need notifications" };
+    
+    const userIds = subscriptionsToNotify.map(s => s.hemocioneId).filter(Boolean) as string[];
```

ESLint

[error] 'userIds' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)